oct/cli/package/common_options.py

```diff
@@ -14,21 +14,29 @@ def package_options(func):
     :return: decorated CLI command
     """
     return option(
-        '--update/--upgrade',
-        '-d/-g',
-        'update_current_stage',
-        default=False,
-        help='Update stage or upgrade to next stage.  [default: upgrade]',
+        '--upgrade',
```

Comment:
You can shoot yourself in the foot with `--stage` but if you follow convention you'll be fine. Convention was all that ever held together the other AMIs as well so I think it's OK.